src/LiveApi.js

```diff
@@ -13,6 +13,8 @@ let WebSocket = typeof window !== 'undefined' ? window.WebSocket : MockWebSocket
 const shouldIgnoreError = error =>
     error.message.includes('You are already subscribed to') ||
         error.message.includes('Input validation failed: forget');
+const shouldNotThrow = json => 
+    json.error.code === 'SelfExclusion' && json.msg_type === 'authorize';
 
```

Comment:
@nuruddeensalihu I highly doubt so, if we do not return error code, how do we even handle error? Did you confirm or can you point me to the code that remove/omit error code?

I am pretty sure it does return error code, chances are the structure is too weird that you misunderstood, check here 

https://github.com/binary-com/binary-next-gen/blob/master/src/_trade/saga/LifeCycleSaga.js#L57